wpiutil/src/main/java/edu/wpi/first/util/CircularBuffer.java

```diff
@@ -150,9 +154,6 @@ void resize(int size) {
 
   /** Sets internal buffer contents to zero. */
   public void clear() {
```

Comment:
This could use `m_data[i] = null` instead of 0 from before so that objects get garbage collected.